tool/lib/commands/fix_goldens.dart

```diff
@@ -0,0 +1,101 @@
+// #!/bin/bash -e
+// RUN_ID=$1
+import 'dart:async';
+import 'dart:io';
+
+import 'package:args/command_runner.dart';
+import 'package:io/io.dart';
+import 'package:path/path.dart' as path;
+
+import '../utils.dart';
+
+class FixGoldensCommand extends Command {
+  FixGoldensCommand() {
+    argParser.addOption(
+      'run-id',
+      help: 'The ID of the workflow run where the goldens are failing. '
+          'e.g.https://github.com/flutter/devtools/actions/runs/<run-id>/job/16691428186',
+      valueHelp: '12345',
+      mandatory: true,
+    );
+  }
+  @override
+  String get description =>
+      'A helper script for downloading and applying golden fixes, when they are broken.';
+
+  @override
+  String get name => 'fix-goldens';
+
+  final processManager = ProcessManager();
+
+  @override
+  FutureOr? run() async {
+    // Change the CWD to the repo root
+    Directory.current = pathFromRepoRoot("");
+
+    final runId = argResults!['run-id']!;
+    final tmpDownloadDir = Directory(
+      path.join('.tmp${DateTime.now().millisecondsSinceEpoch}'),
+    );
+    tmpDownloadDir.createSync();
+    try {
+      print('Downloading the artifacts to ${tmpDownloadDir.path}');
+      await processManager.runProcess(
+        CliCommand.from('gh', [
+          'run',
+          'download',
+          runId,
+          '-p',
+          '*golden_image_failures*',
+          '-D',
+          tmpDownloadDir.path,
+        ]),
```

Comment:
trailing comma. We should make sure the trailing commas lint is applied to the tool/ directory

Reply:
@kenzieschmoll which lint is this? Currently tool has `require_trailing_commas`